website/static/js/addProjectPlugin.js

```diff
@@ -35,6 +35,13 @@ var AddProject = {
         self.newProjectDesc = m.prop('');
         self.newProjectCategory = m.prop(self.defaultCat);
         self.newProjectTemplate = m.prop('');
+        self.checkedInstitutions = {};
+        self.newProjectInstitutions = window.contextVars.currentUser.institutions.map(
```

Comment:
Minor:  I still think this should be refactored to be passed as an option, but in the interest of getting this onto staging soon, this is ok. That said, you could easily future-proof by doing

```js
self.institutions = options.institutions || window.contextVars.currentUser.institutions || [];
```

This will make the future refactor easier.